sources/org.osbuild.curl

```diff
@@ -86,6 +87,32 @@ SCHEMA = """
 """
 
 
+def _gen_curl_download_config(items: Dict, config_path: pathlib.Path):
```

Comment:
It looks like this made it in accidentially via https://github.com/osbuild/osbuild/pull/1550/commits/3679abc0f963569ab96d94b3921c863b32a9f56f - I will remove it

Reply:
Fixed